llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -299,6 +299,10 @@ them.
  Allow :program:`llvm-objcopy` to remove sections even if it would leave invalid
  section references. Any invalid sh_link fields will be set to zero.
 
+.. option:: --change-section-lma \*{+-}<val>
+
+ Shift LMA of non-zero-sized sections in the program header table by ``<val>``
```

Comment:
```suggestion
 Shift LMA of non-zero-sized segments by ``<val>``
```
How about this? Sections aren't in the program header table, only program headers (aka segments, which are in turn implicitly in the program header table).

Reply:
Added this suggestion to **Fixup3**